### PR TITLE
Multiple accordion items support added.

### PR DESCRIPTION
**Pull request by mfendeksilverstripe**
Multiple accordion items support added. Change is backwards compatible.

### CODE REVIEW

File: src/models/AccordionItem.php [hunk 1]
```diff
@@ -16,19 +17,49 @@ class AccordionItem extends DataObject
...
+    ];
+
     private static $default_sort = 'SortOrder ASC';
 
+    /**
+     * @return RequiredFields
+     */
+    public function getCMSValidator()
+    {
+        return new RequiredFields(['Group']);
```
Comment (Firesphere) [inline src/models/AccordionItem.php]: I wouldn't make it required, if I don't want to use it, why should I be forced to?

File: src/models/AccordionItem.php [hunk 2]
```diff
@@ -16,19 +17,49 @@ class AccordionItem extends DataObject
     private static $db = [
         'Title'     => 'Varchar(255)',
         'Content'   => 'HTMLText',
+        'Group'     => 'Int(1)',
```
Comment (Firesphere) [inline src/models/AccordionItem.php]: Group is a reserved keyword for the actual `Groups` of members

File: src/models/AccordionItem.php [hunk 3]
```diff
@@ -16,19 +17,49 @@ class AccordionItem extends DataObject
...
+     * @return FieldList
+     */
     public function getCMSFields()
     {
         $fields = parent::getCMSFields();
-        $fields->removeByName(['PageID', 'SortOrder']);
+
+        /** @var $group NumericField */
+        $group = $fields->fieldByName('Root.Main.Group');
+        $group->setDescription('All items that share the same accordion have to have the same accordion ID, accordion shortcode is [accordion,id=ID]');
```
Comment (Firesphere) [inline src/models/AccordionItem.php]: Just an ID is very unclear. At the least, give a list of the ID's currently set/available on this page.

File: templates/AccordionItems.ss [hunk 4]
```diff
@@ -1,15 +1,17 @@
-<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
-    <% loop $AccordionItems %>
+<div class="panel-group" id="accordion-{$AccordionId}" role="tablist" aria-multiselectable="true">
+    <% loop $AccordionItems.Filter('Group', $AccordionId) %>
```
Comment (Firesphere) [inline templates/AccordionItems.ss]: Why not filter in the parser instead of in the template?

File: src/models/AccordionItem.php (hunk 1 above)
Comment (mfendeksilverstripe) [reply src/models/AccordionItem.php]: It's removed now.

File: src/models/AccordionItem.php (hunk 2 above)
Comment (mfendeksilverstripe) [reply src/models/AccordionItem.php]: Renamed to `AccordionID`.

File: src/models/AccordionItem.php (hunk 3 above)
Comment (mfendeksilverstripe) [reply src/models/AccordionItem.php]: Comma separated list of Accordion ids added to description.

File: templates/AccordionItems.ss (hunk 4 above)
Comment (mfendeksilverstripe) [reply templates/AccordionItems.ss]: Filtering moved to parser.

File: src/models/AccordionItem.php [hunk 5]
```diff
@@ -7,28 +7,61 @@
...
  */
 class AccordionItem extends DataObject
 {
     private static $db = [
-        'Title'     => 'Varchar(255)',
-        'Content'   => 'HTMLText',
-        'SortOrder' => 'Int',
+        'Title'         => 'Varchar(255)',
+        'Content'       => 'HTMLText',
+        'AccordionID'   => 'Int(1)',
```
Comment (Firesphere) [inline src/models/AccordionItem.php]: Please don't use `ID` at the end, it confuses with a `has_one` relation. I'd suggest go with `Set`?

Comment (mfendeksilverstripe) [reply src/models/AccordionItem.php]: Renamed to `AccordionSet`.

**Pull request merged**